lib/local_state.py

```diff
@@ -280,8 +284,12 @@ def get_key_path_from_name(cls, keyname):
     Returns:
       A str that indicates where the private key can be found.
     """
-    return cls.LOCAL_APPSCALE_PATH + keyname + ".key"
-
+    for local_path in cls.VALID_KEY_PATHS:
+      key_file_path = local_path + keyname + ".key"
+      if (os.path.isfile(key_file_path)):
```

Comment:
Change this line to: 
`if os.path.isfile(key_file_path):`

The external parentheses are not needed.